pkg/asset/tls.go

```diff
@@ -82,6 +82,7 @@ func newAPIKeyAndCert(caCert *x509.Certificate, caPrivKey *rsa.PrivateKey, altNa
 		"kubernetes.default",
 		"kubernetes.default.svc",
 		"kubernetes.default.svc.cluster.local",
+		"localhost",
```

Comment:
Hmm. I remember seeing a bunch of SAN errors because _something_ was connecting with TLS over localhost. For the life of me I can't remember what it was though 😅  I'll try to dig deeper on this.